code/__DEFINES/dcs/signals/signals_hud.dm

```diff
@@ -2,4 +2,7 @@
 #define COMSIG_HUD_EYE_CHANGED "hud_eye_changed"
 /// Sent from /datum/hud/proc/eye_z_changed() : (old_offset, new_offset)
 #define COMSIG_HUD_OFFSET_CHANGED "hud_offset_changed"
-
+/// Sent from /atom/movable/screen/lobby/button/collapse/proc/collapse_buttons()
+#define COMSIG_HUD_LOBBY_COLLAPSED "hud_lobby_collapsed"
+/// Sent from /atom/movable/screen/lobby/button/collapse/proc/expand_buttons()
```

Comment:
```suggestion
/// Sent from /atom/movable/screen/lobby/button/collapse/proc/collapse_buttons() : ()
#define COMSIG_HUD_LOBBY_COLLAPSED "hud_lobby_collapsed"
/// Sent from /atom/movable/screen/lobby/button/collapse/proc/expand_buttons() : ()
```
format is source : (arguments, ignoring source), helps people know what they get from the signal